src/annotator/test/config-test.js

```diff
@@ -185,33 +185,43 @@ describe('annotator.config', function() {
     });
   });
 
-  it("extracts the annotation query from the parent page's URL", function() {
+  it("extracts the direct-linked annotation ID from the parent page's URL", function() {
     configFrom(fakeWindow());
 
-    assert.calledOnce(fakeExtractAnnotationQuery);
-    assert.calledWithExactly(fakeExtractAnnotationQuery, 'LOCATION_HREF');
+    assert.calledOnce(fakeExtractAnnotationQuery.annotations);
+    assert.calledWithExactly(
+      fakeExtractAnnotationQuery.annotations, 'LOCATION_HREF');
   });
 
-  context('when extractAnnotationQuery() returns an object', function() {
+  context("when there's a direct-linked annotation ID", function() {
     beforeEach(function() {
-      fakeExtractAnnotationQuery.returns({foo: 'bar'});
+      fakeExtractAnnotationQuery.annotations.returns('ANNOTATION_ID');
     });
 
-    it('blindly adds the properties of the object to the config', function() {
-      assert.equal(configFrom(fakeWindow()).foo, 'bar');
+    it('adds the annotation ID to the config', function() {
+      assert.equal(configFrom(fakeWindow()).annotations, 'ANNOTATION_ID');
     });
+  });
 
-    specify('settings from extractAnnotationQuery override others', function() {
```

Comment:
I deleted this "override others" test. I don't think it's important and the overriding will be disappearing in a following PR anyway.